frontend/packages/console-shared/src/components/dashboard/status-card/OperatorStatusBody.tsx

```diff
@@ -43,11 +45,12 @@ export const OperatorsSection: React.FC<OperatorsSectionProps> = ({
       ) : (
         !operatorsHealthy &&
         sortedOperatorStatuses.map((operatorStatus) => (
-          <AsyncComponent
+          <Row
             key={operatorStatus.operators[0].metadata.uid}
+            Component={Component}
```

Comment:
Just wondering, both `Row` and `Component` get passed here, so why do we assign `Component` to `Row` here, instead of expecting the `Row` itself to have it already?